blog/content/post/2021-12-12-optiplex-txt.md

```diff
@@ -0,0 +1,222 @@
+---
+title: A new source of trust for your platform - Intel TXT for Dasharo
```

Comment:
I would be glad to change this title so Dasharo is not slave to Intel TXT.  `Dasharo with Intel TXT support` ?

Reply:
Done